tests/integration/security/main_test.go

```diff
@@ -49,6 +49,25 @@ func setupConfig(ctx resource.Context, cfg *istio.Config) {
 	cfg.ControlPlaneValues = `
 values:
   pilot: 
+    jwksResolverExtraRootCA: |
+      MIIDRTCCAi2gAwIBAgIUBeTBvkDJcwG0pIYWP/VphzEGWVEwDQYJKoZIhvcNAQEL
```

Comment:
Consider adding a comment on how this cert is generated, similar to the following comment in the PR
#command to generate certificate and a key
  #openssl req \ -x509 \ -nodes \ -newkey rsa:2048 \ -keyout server.key \ -out server.crt \ -days 3650 \

Reply:
sure, thanks for the suggestion !!